.github/workflows/e2e.yml

```diff
@@ -20,6 +20,18 @@ jobs:
         os: [macos-latest, windows-latest, ubuntu-22.04]
         frameworks: ["nextjs", "express", "fastapi"]
         datasources: ["--no-files", "--example-file"]
+        templates: ["streaming", "extractor"]
```

Comment:
Why do we need to iterate templates via matrix if we have one test file per template?

Reply:
Splitting the template make the code easier to read but the gh actions still the same. We can of course create a separated action for each template but just simply adding new template to the matrix still fine for me